app/experimenter/experiments/models.py

```diff
@@ -377,17 +381,38 @@ def has_normandy_info(self):
         return self.normandy_slug or self.normandy_id
 
     @property
-    def normandy_api_recipe_url(self):
-        if self.normandy_id:
-            return settings.NORMANDY_API_RECIPE_URL.format(id=self.normandy_id)
+    def format_dc_normandy_urls(self):
+        # returns a list of dictionaries containing D.C. and Normandy
+        # urls for the main normandy id and other normandy ids if they exist
+        normandy_recipe_url = settings.NORMANDY_API_RECIPE_URL
+        delivery_console_url = settings.DELIVERY_CONSOLE_RECIPE_URL
 
-    @property
-    def delivery_console_recipe_url(self):
         if self.normandy_id:
-            return settings.DELIVERY_CONSOLE_RECIPE_URL.format(
-                id=self.normandy_id
+            urls = []
```

Comment:
You can collapse these two lines together...

    urls = [{
        ....
    }]

Reply:
Alternatively you could change this whole function signature to always return a list

    urls = []
    if self.normandy_id:
        ....
    if self.other_normandy_ids:
        ....
    return urls

then the return type is always the same and you can just say

    {% for url in experiment.format.... %}
    {% %}

instead of needing an `if` check?

Reply:
Ah I forgot that templates will just silently fail on that, but still, if this ever gets used somewhere else it might be nice if the return type is always a list..... 🤔

Reply:
Yes I updated it to return an empty list either way. I considered that initially... not sure why I returned none instead...